sources/packages/api/src/route-controllers/user/user.controller.ts

```diff
@@ -120,4 +121,16 @@ export class UserController extends BaseController {
       throw error;
     }
   }
+
+  @Get("/institutionUser")
+  async institutionDetail(
+    @UserToken() userToken: IUserToken,
+  ): Promise<InstitutionUserDto> {
+    const user = await this.service.getActiveUser(userToken.userName);
+    const institutionUser = new InstitutionUserDto();
+    institutionUser.userEmail = user?.email;
```

Comment:
At this point, the user must have a value and there is no need for a null check.

Reply:
Null check has been removed